File: 프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.java
import java.util.Arrays;

class Solution {
    public int solution(int[][] dots) {
        int[] xs = {dots[0][0], dots[1][0], dots[2][0], dots[3][0]};
        int[] ys = {dots[0][1], dots[1][1], dots[2][1], dots[3][1]};

        Arrays.sort(xs);
        Arrays.sort(ys);

        int width = xs[3] - xs[0];
        int height = ys[3] - ys[0];

        return width * height;
    }
}

File: 프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.js
function solution(dots) {
    const xs = dots.map(dot => dot[0]);
    const ys = dots.map(dot => dot[1]);

    const width = Math.max(...xs) - Math.min(...xs);
    const height = Math.max(...ys) - Math.min(...ys);

    return width * height;
}